blockchain/indexers/manager.go

```diff
@@ -47,10 +47,18 @@ func dbPutIndexerTip(dbTx database.Tx, idxKey []byte, hash *chainhash.Hash, heig
 
 // dbFetchIndexerTip uses an existing database transaction to retrieve the
 // hash and height of the current tip for the provided index.
-func dbFetchIndexerTip(dbTx database.Tx, idxKey []byte) (*chainhash.Hash, int32, error) {
+func dbFetchIndexerTip(dbTx database.Tx, indexerPtr *Indexer) (*chainhash.Hash, int32, error) {
```

Comment:
Since `Indexer` is an interface this doesn't need to be a pointer.

Reply:
thanks.  With the new `StartBlock()` method added, this code has been reverted to its original form.